apps/backoffice-v2/src/pages/Entity/components/Case/components/CaseOverview/CaseOverview.tsx

```diff
@@ -29,25 +29,29 @@ export const CaseOverview = ({ processes }: { processes: string[] }) => {
     workflow?.context?.pluginsOutput?.riskEvaluation?.riskIndicatorsByDomain ??
       workflow?.context?.pluginsOutput?.risk_evaluation?.riskIndicatorsByDomain ??
       {},
-  )?.map(([domain, riskIndicators]) => {
-    const tab = camelCase(domain.toLowerCase());
-    const isValidCaseTab = CaseTabs.includes(tab);
+  )
+    ?.map(([domain, riskIndicators]) => {
+      const domainTitle = domain ?? '';
+      const tabEntry = Object.entries(TabToLabel).find(([_, label]) => label === domainTitle);
+      const tab = tabEntry ? tabEntry[0] : camelCase(domainTitle.toLowerCase());
+      const isValidCaseTab = CaseTabs.includes(tab as any);
```

Comment:
Let's not add `any` type
```suggestion
      const isValidCaseTab = CaseTabs.includes(tab as keyof typeof CaseTabs);
```